src/components/common/students/students-list/myFilter.tsx

```diff
@@ -0,0 +1,23 @@
+import { FilterChips } from "@/components/design-system/chips";
+import { FilterProps } from "@/types/students/students-list-types";
+import { useState, useEffect } from "react";
+
+export const Filters = ({ filterDetails, onFilterChange, clearFilters }: FilterProps) => {
+    const [selectedFilters, setSelectedFilters] = useState<string[]>([]);

```

Comment:
Depends on test cases but if this thing can be done via zustand, then you don't beed to pass setters in filter chips